Presets/Dawntrail/Raids/The Arcadion 1 Savage.md

```diff
@@ -110,3 +110,9 @@ This displays red when a tile will receive damage and black when a tile will bre
s":11.0,"color":4278190335,"fillIntensity":0.3,"originFillColor":838861055,"endFillColor":838861055,"refActorNPCNameID":13071,"refActorRequireCast":true,"refActorCastId":[39633],"refActorUseCastTime":true,"refActorCastTimeMin":16.0,"refActorCastTimeMax":19.0,"refActorUseOvercast":true,"refActorComparisonType":6,"includeRotation":true,"FaceMe":true,"refActorTetherTimeMin":0.0,"refActorTetherTimeMax":0.0},{"Name":"Line1","type":3,"refY":20.0,"radius":3.0,"color":4278190335,"fillIntensity":0.3,"originFillColor":838861055,"endFillColor":838861055,"refActorNPCNameID":13071,"refActorRequireCast":true,"refActorCastId":[38026],"refActorUseCastTime":true,"refActorCastTimeMax":15.0,"refActorUseOvercast":true,"refActorComparisonType":6,"includeRotation":true,"refActorTetherTimeMin":0.0,"refActorTetherTimeMax":0.0,"FillStep":2.0},{"Name":"Line2","type":3,"refY":20.0,"radius":3.0,"color":4278190335,"fillIntensity":0.3,"originFillColor":838861055,"endFillColor":838861055,"refActorNPCNameID":13071,"refActorRequireCast":true,"refActorCastId":[38028],"refActorUseCastTime":true,"refActorCastTimeMax":15.0,"refActorUseOvercast":true,"refActorComparisonType":6,"includeRotation":true,"refActorTetherTimeMin":0.0,"refActorTetherTimeMax":0.0,"FillStep":2.0},{"Name":"Line3","type":3,"refY":20.0,"radius":3.0,"color":4278190335,"fillIntensity":0.3,"originFillColor":838861055,"endFillColor":838861055,"refActorNPCNameID":13071,"refActorRequireCast":true,"refActorCastId":[38030],"refActorUseCastTime":true,"refActorCastTimeMax":16.0,"refActorUseOvercast":true,"refActorComparisonType":6,"includeRotation":true,"refActorTetherTimeMin":0.0,"refActorTetherTimeMax":0.0,"FillStep":2.0},{"Name":"Line4","type":3,"refY":20.0,"radius":3.0,"color":4278190335,"fillIntensity":0.3,"originFillColor":838861055,"endFillColor":838861055,"refActorNPCNameID":13071,"refActorRequireCast":true,"refActorCastId":[38032],"refActorUseCastTime":true,"refActorCastTimeMax":17.0,"refActorUseOvercast":true,"refActorComparisonType":6,"includeRotation":true,"refActorTetherTimeMin":0.0,"refActorTetherTimeMax":0.0,"FillStep":2.0},{"Name":"Line5","type":3,"refY":20.0,"radius":3.0,"color":4278190335,"fillIntensity":0.3,"originFillColor":838861055,"endFillColor":838861055,"refActorNPCNameID":13071,"refActorRequireCast":true,"refActorCastId":[38034],"refActorUseCastTime":true,"refActorCastTimeMin":15.0,"refActorCastTimeMax":18.0,"refActorUseOvercast":true,"refActorComparisonType":6,"includeRotation":true,"refActorTetherTimeMin":0.0,"refActorTetherTimeMax":0.0,"FillStep":2.0},{"Name":"Line6","type":3,"refY":33.0,"radius":3.0,"color":4278190335,"fillIntensity":0.3,"originFillColor":838861055,"endFillColor":838861055,"refActorNPCNameID":13071,"refActorRequireCast":true,"refActorCastId":[39632],"refActorUseCastTime":true,"refActorCastTimeMin":16.0,"refActorCastTimeMax":19.0,"refActorUseOvercast":true,"refActorComparisonType":6,"includeRotation":true,"refActorTetherTimeMin":0.0,"refActorTetherTimeMax":0.0,"FillStep":2.0}]}
```

Comment:
I would update the name to have a space between `Raining` and ` Cats`:
```json
"Name":"M1S Raining Cats"
```

Reply:
Fixed.
Thank you.